src/collection-proxy.js

```diff
@@ -518,6 +518,17 @@ class CollectionProxy {
     return this._enqueue('unshift', null, items)
   }
 
+  /**
+  * Returns the average of all collection items
+  *
+  * @returns {Number}
+  * */

```

Comment:
Can you please indent the comment with one more space? The asterisks should align with the first asterisk from the starting comment. Please have a look at the comments of other methods.